src/constants/network.ts

```diff
@@ -48,6 +48,54 @@ export const nativeTokens: Record<string, Token[]> = {
       protocol: "",
     },
   ],
+  fetchhub: [
```

Comment:
Don't define it here, use that endpoint.
https://dorado-api.dezswap.io/v1/tokens